plugins/verifier/cosign/cosign.go

```diff
@@ -96,7 +96,7 @@ func VerifyReference(args *skel.CmdArgs, subjectReference common.Reference, refe
 	if err != nil {
 		return nil, err
 	}
-	verifierType := ""
+	verifierType := input.Config.Name
```

Comment:
wondering why cosign is different to other verifiers?  input.Config.Name vs input.Name.

Reply:
yeah that's a good question. When the external plugin is invoked, the executor will actually provide the store config in the JSON input. Other plugins don't need to use except for the old cosign implementation which needed the `useHTTP` flag in the store config to configure its own remote fetcher API. Cosign uses Crane and not ORAS. Since then, we've removed any online remote fetching in the cosign verifier and switched to use the ORAS store. This is a remnant from those days. There was a `StoreConfig` and a `Config` field which means we had to specify explicitly the `Config` in the accessor. Other plugins don't need this since there's only one field `Config`, thus Go will implicitly know that we are referring to `Config`.